internal/manifests/targetallocator/poddisruptionbudget.go

```diff
@@ -26,17 +27,20 @@ import (
 )
 
 func PodDisruptionBudget(params manifests.Params) (*policyV1.PodDisruptionBudget, error) {
-	if params.OtelCol.Spec.TargetAllocator.AllocationStrategy != v1alpha1.OpenTelemetryTargetAllocatorAllocationStrategyConsistentHashing {
-		params.Log.Info("current allocation strategy not compatible, skipping podDisruptionBudget creation")
-		return nil, nil
-	}
-
-	// defaulting webhook should always set this, but if unset then return nil.
+	// defaulting webhook should set this if the strategy is compatible, but if unset then return nil.
 	if params.OtelCol.Spec.TargetAllocator.PodDisruptionBudget == nil {
 		params.Log.Info("pdb field is unset in Spec, skipping podDisruptionBudget creation")
```

Comment:
can we change this to a debug log as well?

Reply:
Sure!

Reply:
V1 is correct? Or maybe this is V2?

Reply:
I think we do V4

Reply:
done!